packages/types/src/pricing/common/price-rule.ts

```diff
@@ -57,23 +58,28 @@ export interface PriceRuleDTO {
  */
 export interface CreatePriceRuleDTO {
   /**
-   * The ID of the price rule.
+   * The ID of the associated price set.
    */
-  id: string
+  price_set_id?: string
   /**
    * The ID of the associated price set.
    */
-  price_set_id: string
+  price_set?: string | PriceSetDTO
   /**
    * The ID of the associated rule type.
    */
-  rule_type_id: string
+  rule_type_id?: string
+  /**
+   * The ID of the associated rule type.
+   */
+  rule_type?: string | RuleTypeDTO
   /**
    * The value of the price rule.
    */
   value: string
   priority?: number
-  price_set_money_amount_id: string
+  price_set_money_amount_id?: string
+  price_set_money_amount?: string | PriceSetMoneyAmountDTO
```

Comment:
We should add TSDocs for these three, not sure why they're missing 😅 For `priority` and `price_set_money_amount_id` you can use the same TSDocs in `PriceRuleDTO`. I would assume `price_set_money_amount` is similar to `price_set_money_amount_id` but could be something like `The ID or object of the price set money amount`.